src/pages/docs/step-ca/provisioners.mdx

```diff
@@ -609,6 +609,38 @@ Bottom line, the OAuth flow implemented in `step` and `step-ca` is widely vetted
   </div>
 </Alert>
 
+#### Device Authorization Grant Flow
+
+The Device Authorization Grant flow is an OAuth 2.0 extension that enables OAuth for devices with no browser 
+or limited input capability to obtain an access token.
+The OIDC provisioner supports the Device Authorization Grant flow, however,
+it is not always the default flow for a given OIDC provider.
+
+The Google OIDC provider initially defaulted to a different flow - the Out Of Band (OOB) flow which has since been deprecated. 
+`step-ca` OIDC provisioners configured to use Google OIDC will still default to the OOB flow (until October 3rd, 2022).
+Users can specify the proper OAuth 2.0 flow by supplying flags and environment variables.
+
+To use the the default OAuth 2.0 flow for input constrained devices (for your OIDC provider) use:
+
+```shell-session
+$ step ca certificate foo foo.crt foo.key --console
+```
+
+For `step oauth`, specify a non-default OAuth 2.0 input constrained flow with the `--console-flow device` flag and value:
+
+```shell-session
+$ step oauth --client-id <my-client-id> --client-secret <my-client-secret> --console-flow device
+```
+
+For any other `step` command, specify a non-default OAuth 2.0 input constrained flow with the `STEP_CONSOLE_FLOW=device` environment variable ahead of the command:
+
+```shell-session
+$ STEP_CONSOLE_FLOW=device step ca certificate foo foo.crt foo.key
+```
+
```

Comment:
This may also be out of scope for this page, since we cover `STEP_` variables in the Step CLI docs. But, is there a special reason people might want to know they can use the env variable option here?

Reply:
`step ca xxx` do not accept the console_flow flag. Only `step oauth` accepts that flag. So you need to use the env var for `step ca xxx` if you want to change the default flow.